### DISCUSSION TITLE
No interval data for lapped cars

### DISCUSSION THREAD

**Discussion by alfietaylor15** [br-g/openf1#107] [General]
Pulling some data out and not seeing interval data from Alonso down (for singapore)
Is this just because they were lapped or is it a bug?

![image](https://github.com/user-attachments/assets/a07a0d5d-8e54-4d17-ac23-40f2e34c80bd)

**Comment by bpougala** [comment]
The intervals are calculated using the timings over the last completed lap at a given time, which is the lap in which the leading cars are. For that reason, lapped cars do not have an interval calculated for them.
